src/Service/ArtPictureUploader.php

```diff
@@ -31,7 +31,8 @@ public function upload(UploadedFile $file): string
 
     public function edit(UploadedFile $file, string $oldFile): string
     {
-        unlink($oldFile);
+
+        unlink($_SERVER['DOCUMENT_ROOT'] . $oldFile);
```

Comment:
As seen together find another way than DOCUMENT_ROOT to unlink your old file

Reply:
change by getcwd() . $oldFile